### PR TITLE
Fix/text changes

### PR DESCRIPTION
**Pull request by karlbaumhauer**

### CODE REVIEW

File: src/components/cms/bnb-cms-contact/template.ts [hunk 1]
```diff
@@ -23,8 +23,8 @@ export const template = (
 				></bnb-input>
 				<bnb-input
 					required
-					label="Email"
-					placeholder="Email (Bsp: kontakt@brandnewbundestag.de)"
+					label="email"
```
Comment (koepferd) [inline src/components/cms/bnb-cms-contact/template.ts]: Is the label really supposed to be lowercase?

File: src/components/cms/bnb-cms-footer/template.ts [hunk 2]
```diff
@@ -50,7 +50,7 @@ export const template = (data: BnbCmsFooterProps): HTMLFragment => {
 				</div>
 				<div class="footer-links--middle">
 					<bnb-paragraph class="headline"
-						>So erreichst Du uns:</bnb-paragraph
+						>So erreichst duuns:</bnb-paragraph
```
Comment (koepferd) [inline src/components/cms/bnb-cms-footer/template.ts]: Typo

File: src/components/cms/bnb-cms-sub-stage/scaffolding/bnb-cms-sub-stage.hbs [hunk 3]
```diff
@@ -4,7 +4,7 @@ heading="Aktuelles von Brand New Bundestag" image='{"src":"../../_assets/img/sta
     {"src":"../../_assets/img/stage_1024.jpeg", "width":1024},
     {"src":"../../_assets/img/stage_1440.jpeg", "width":1440},
     {"src":"../../_assets/img/stage_1920.jpeg", "width":1920}]}'
-data-resources="[{paths: ['components/cms/bnb-cms-sub-stage/index.js']}]">Hier findest Du alle unsere News
+data-resources="[{paths: ['components/cms/bnb-cms-sub-stage/index.js']}]">Hier findest dualle unsere News
```
Comment (koepferd) [inline src/components/cms/bnb-cms-sub-stage/scaffolding/bnb-cms-sub-stage.hbs]: Typo

File: src/components/cms/bnb-cms-contact/template.ts (hunk 1 above)
Comment (karlbaumhauer) [reply src/components/cms/bnb-cms-contact/template.ts]: its a hidden label and only for screenreaders

**Pull request merged**